Destiny.ScrimTracker.Logic/Models/MatchTeam.cs

```diff
@@ -0,0 +1,11 @@
+using System.Collections;
+using System.Collections.Generic;
+
+namespace Destiny.ScrimTracker.Logic.Models
+{
+    public class MatchTeam
+    {
+        public IEnumerable<GuardianMatchResult> Guardians { get; set; }
```

Comment:
This feels awkward to me... but I'm not sure why. My first thought was "this should be linking to guardians, not the results! But I don't think that's right either...

If you have a match, to get a list of guardians that participated you'd need to go:

`Match -> Match Team -> Match Result -> Guardian` which is a lot of traversal. Maybe that's okay? What do you think?

Reply:
That is a lot of traversal, but i wasn't sure how else to better track all this information. It doesn't seem like _too_ much of a bad thing to me? We could also add a match ID to match result to maintain the two way relationship?

Reply:
Yea let's def add the ID to the `MatchResult`